main/getting-started/intro-zoe.md

```diff
@@ -1,358 +1,286 @@
 # Introduction to Zoe
 
-**Note:** Before reading this, you should be familiar with [ERTP basics](/getting-started/ertp-introduction/). 
-
 ::: tip Alpha status
 Zoe is currently at Alpha status. It has not yet been
 formally tested or hardened. It is not yet of production quality.
 :::
 
-This is an introduction to Zoe, Agoric's smart contract framework. 
-The [Zoe Guide](/zoe/guide/) has additional information, 
-including concepts not covered here. 
+## What is Zoe?
 
-You can run different types of smart contracts on Zoe. In this document, 
-for the most part we focus on Agoric’s Atomic Swap demo contract, but 
-sometimes refer to how Zoe works with other contract types.
+Zoe is Agoric's smart contract framework. You should use Zoe to:
 
-## What is Zoe?
+* **Run your code on-chain**
+* **Mint new digital assets**
+* **Credibly trade assets**
+
+Zoe relies on [ERTP](./ertp-introduction.md), our token standard for fungible and non-fungible
+tokens. 
```

Comment:
```suggestion
Zoe relies on Agoric's [*Electronic Rights Transfer Protocol (ERTP)*](./ertp-introduction.md), our token standard.
```

Reply:
I think it's not important to specify the full name in the Zoe intro, since it doesn't help explain anything and we also don't use the full name often. ERTP and Electronic Rights Transfer Protocol are not great names, so I'd rather not spend much time on them.

Reply:
On the one hand, yes. On the other, it's generally considered good writing practice to always expand an acronym on first usage, and there are some people who get irked if a doc doesn't do so (the always exception is if it's either effectively become a word in its own right (ATM for the clear reason, NAACP for less obvious reasons) or if one can reasonably assume everyone knows what it stands for)

Reply:
Yeah, good point. I suppose there may readers who will feel like they are missing something, even though there is nothing to miss.